src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/inmemoryrepo/InMemoryCourseRepositoryTest.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.inmemoryrepo
```

Comment:
This should be squashed into the previous commit where the `InMemoryCourseRepositoryTest` is introduced, rather than live on `Add flyway and flyway migration for CourseEntity`